spec/classes/master_maintenance_spec.rb

```diff
@@ -110,6 +110,32 @@
         })
       }
     end
+
+    context 'disabled with boolean' do
+      let(:facts) do
+        { :puppet_reportdir => '/var/lib/puppet/reports', }
+      end
+      let(:params) do
+        { :reportdir_purge_enabled => false, }
+      end
+
+      it { should contain_class('puppet::master::maintenance') }
+
+      it { should_not contain_cron('purge_old_puppet_reports') }
```

Comment:
this should be checking `cron { 'filebucket_cleanup'` as it is under the description of 'clientbucket cleanup'. Also instead of two separate contexts you can use an array of `[false,'false']`. Here's an example

https://github.com/ghoneycutt/puppet-module-puppet/blob/master/spec/classes/dashboard_maintenance_spec.rb#L157-178